pybossa/api/task_run.py

```diff
@@ -70,14 +72,23 @@ def _preprocess_post_data(self, data):
         project_id = data['project_id']
         self.check_can_post(project_id, task_id)
         preprocess_task_run(project_id, task_id, data)
-        info = data['info']
-        path = "{0}/{1}/{2}".format(project_id, task_id, current_user.id)
         if with_encryption:
```

Comment:
Just wanted to confirm if moving lines 73, 74 into this "if" block was intended, as they will only be executed now if `with_encryption`.

```python
info = data['info']
path = "{0}/{1}/{2}".format(project_id, task_id, current_user.id)
```

Reply:
info and path generated were used only with_encryption true, hence its safe to set it only when needed. Refer original code [here](https://github.com/bloomberg/pybossa/blob/master/pybossa/api/task_run.py#L73-L80)